### PR TITLE
Add noop zuul job to test zuul integration

### PR DESCRIPTION
**Pull request by cjeanner**
This PR has:
- [x] Appropriate testing (molecule, python unit tests)
- [x] Appropriate documentation (README in the role, main README is up-to-date)

### CODE REVIEW

**Comment by cjeanner** [comment]
recheck

no `/` for zuul.

**Comment by raukadah** [comment]
check-rdo

**Comment by cjeanner** [comment]
recheck

**Comment by raukadah** [comment]
check-rdo

File: .zuul.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+- job:
+    name: molecule
+    nodeset: ansible-cloud-centos-9-stream-tiny
```
Comment (raukadah) [inline .zuul.yaml:4]: ansible-cloud-centos-9-stream-tiny is a label. From label, we need to define the nodeset.
```
    nodeset:
      nodes:
        name: primary
        label: ansible-cloud-centos-9-stream-tiny
```
Can we try with this?

**Comment by cjeanner** [comment]
/hold

need to make zuul required.

**Comment by cjeanner** [comment]
Enforcing Zuul pipeline is here: https://github.com/openshift/release/pull/36925

**Pull request merged**